src/Scaffolding/VS.Web.CG.Mvc/BlazorIdentity/BlazorIdentityHelper.cs

```diff
@@ -101,5 +101,23 @@ internal static CodeSnippet[] ApplyIdentityChanges(CodeSnippet[] filteredChanges
 
             return filteredChanges;
         }
+
+        internal static string BlazorIdentityReadmeFileName = "Scaffolding-README.md";
+        internal static string BlazorIdentityReadmeString =
+@"Blazor Identity scaffolding has completed successfully.
+
+For setup and configuration information, see https://learn.microsoft.com/en-us/aspnet/core/blazor/security.
```

Comment:
Should this be an aka.ms or fwlink instead?

Reply:
yup should be, was being lazy. Updated